pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -122,19 +122,16 @@
 
   public BaseBrokerRequestHandler(PinotConfiguration config, RoutingManager routingManager,
       AccessControlFactory accessControlFactory, QueryQuotaManager queryQuotaManager, BrokerMetrics brokerMetrics,
-      ZkHelixPropertyStore<ZNRecord> propertyStore) {
+      TableCache tableCache) {
     _config = config;
     _routingManager = routingManager;
     _accessControlFactory = accessControlFactory;
     _queryQuotaManager = queryQuotaManager;
     _brokerMetrics = brokerMetrics;
+    _tableCache = tableCache;
 
     _enableCaseInsensitive = _config.getProperty(CommonConstants.Helix.ENABLE_CASE_INSENSITIVE_KEY, false);
-    if (_enableCaseInsensitive) {
-      _tableCache = new TableCache(propertyStore);
```

Comment:
Why was previously the cache only instantiated if case insensitivity was enabled, but now it is regardless? What impact will that have?

Reply:
It's to avoid instantiating the cache in cases it isn't needed. Mayank also called this out above...but it should be fine.

> Just want to call out every deployment will now read the table config for the entire cluster, per broker. I think we do that for controller, and recently added for server.